week1/extra-homework/server.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const http = require('http');
+const sendIndex = require('./sendIndex');
+const sendJS = require('./sendJS');
+const sendImage = require('./sendImage');
+const sendStyle = require('./sendStyle');
+
+function createServer(port) {
+  function handleRequest(req, res) {
```

Comment:
Since it's mostly we are handling the response to be served, or in other words, we are creating a server.
is handle request a descriptive name?

Reply:
You're totally right. Since we are at the server side, we are getting the request from the client and handling(dealing with) the response because `request` is read-only for us whereas `response` is writable. On the other hand documentation calls this function as `requestListener`. I am a bit confused with this callback and not very good at naming functions or variables 😁
Changed to `handleResponse`